lesson3/hometask_argirovatina.rb

```diff
@@ -0,0 +1,31 @@
+module MyEach
+
+  def my_each
+    self.to_a.flatten if self.is_a?(Hash)
+    if block_given?
+      new_array = []
+      current_number = self.count
+      while current_number > 0
+        new_array << yield(self[current_number - 1])
+        current_number -= 1
+      end
+    end
+    p new_array.reverse!
```

Comment:
Why do you output this array here? You custom :each iterator should not do this!